src/storyMap/components/UserStoryMap.js

```diff
@@ -71,6 +74,8 @@ const UserStoryMap = () => {
     )
   );
 
+  const onDeleteSuccess = useCallback(() => navigate('/'), [navigate]);
```

Comment:
question: does this go back to root? I would expect to go back to the Story Maps page.